src/org/thoughtcrime/securesms/ConversationFragment.java

```diff
@@ -856,7 +856,7 @@ public void onQuoteClicked(DcMsg messageRecord) {
       public void onShowFullClicked(DcMsg messageRecord) {
         Intent intent = new Intent(getActivity(), FullMsgActivity.class);
         intent.putExtra(FullMsgActivity.MSG_ID_EXTRA, messageRecord.getId());
-        intent.putExtra(FullMsgActivity.IS_CONTACT_REQUEST, getListAdapter().getChat().isContactRequest());
+        intent.putExtra(FullMsgActivity.BLOCK_LOADING_REMOTE, getListAdapter().getChat().isHalfBlocked());
```

Comment:
was just wondering about loading remote image blocking in iOS. if in doubt, it probably makes sense to mimic behaviour of contact-request here